Assets/Scripts/Gamestate/MatchController.cs

```diff
@@ -101,7 +101,7 @@ void Start()
         // Makes shooting end quickly if testing with 1 player
 #if UNITY_EDITOR
         if (PlayerInputManagerController.Singleton.playerInputs.Count == 1)
-            roundLength = 60f;
+            roundLength = 5f;
```

Comment:
Ah, another reminder that we should have a keybind that skips the round in debug mode...